.pre-commit-config.yaml

```diff
@@ -0,0 +1,19 @@
+repos:
+-   repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.3.0
+    hooks:
+    -   id: trailing-whitespace
+    -   id: end-of-file-fixer
+        exclude: docs/
+    -   id: check-yaml
+    -   id: check-added-large-files
```

Comment:
We can also add `debug-statements`, which ensures no `breakpoint`s are merged in:

```
    - id: debug-statements
```